src/Parser/BloodDeathKnight/CombatLogParser.js

```diff
@@ -28,6 +28,7 @@ class CombatLogParser extends CoreCombatLogParser {
     OssuaryUptime: OssuaryUptime,
     WastedDeathAndDecay: WastedDeathAndDecay,
     RunicPowerWasted: RPWasted,
+    BlooddrinkerTicks: BlooddrinkerTicks,
```

Comment:
These should be named like:
`blooddrinkerTicks: BlooddrinkerTicks,`

This is a pretty ubiquitous naming convention, the capital at the start means it refers to the class itself, whereas the lack indicates that it is an instance of that class.